3Drat/main.js

```diff
@@ -0,0 +1,101 @@
+import * as THREE from 'threejs/three.js';
```

Comment:
the file is not found when running with a live server.

I tried using relative dirs and the browser complains about this instead:

```
http://localhost:5500/threejs/three net::ERR_ABORTED 404 (Not Found)
```

 -- why dont you just use npm and a bundler like snowpack...?